server/auth_test.go

```diff
@@ -76,24 +76,11 @@ func TestJWTTokenMalformed(t *testing.T) {
 	})
 }
 
-// TestJWTToken checks authorization through Authorization header
```

Comment:
was this test incorrect?

Reply:
I don't even remember how I removed it, maybe it was while rebasing. Now it's back